### ISSUE TITLE
Import-Module AzureRM.Netcore fails

### ISSUE THREAD

**Issue by jebalert** [Azure/azure-powershell#6656]
### Description

I'm unable to Import the AzureRM module in PS core. I've downloaded the most recent version.

### Script/Steps for Reproduction

```powershell
Import-Module AzureRM.Netcore 
```

### Module Version

```powershell
Get-Module -Name AzureRM -ListAvailable

This returns nothing.
```

### Environment Data

```powershell
Test-Path $PSVersionTable
False

(PS Core 6.0.1.0, macOS 10.13.1)
```

### Debug Output

```
Import-Module : The following error occurred while loading the extended type data file: 
, /usr/local/share/powershell/Modules/AzureRM.Profile.Netcore/0.12.0/Microsoft.Azure.Commands.Profile.types.ps1xml(85) : Error: Unable to find type [Microsoft.Azure.Commands.Profile.Models.AzureContextConverter].
At line:1 char:1
+ Import-Module AzureRM.Profile.Netcore
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : InvalidOperation: (:) [Import-Module], RuntimeException
+ FullyQualifiedErrorId : FormatXmlUpdateException,Microsoft.PowerShell.Commands.ImportModuleCommand
```

**Comment by cormacpayne** [comment]
@jebalert you will need to run `Get-Module -Name AzureRM.Netcore -ListAvailable` to display the version of `AzureRM.Netcore` installed on your machine. Do you see the above error _only_ when running `Import-Module -Name AzureRM.Netcore`, or do you also see it when running any command without importing first?

**Comment by jebalert** [comment]
@cormacpayne I ran that per the instructions when posting the issue. It returns nothing. However, many Azure specific cmdlets are indeed available to me... connect-azurermaccount requires me to import the module but I’m unable. 

When I run Connect-AzureRMAccount I receive an error saying something along the lines of the “cmdlet is available but could not load the module”.

**Comment by cormacpayne** [comment]
@jebalert `AzureRM` and `AzureRM.Netcore` are two separate modules, and currently the template says to run `Get-Module -Name AzureRM -ListAvailable`, but in your case, you'll need to run `Get-Module -Name AzureRM.Netcore -ListAvailable`

**Comment by jebalert** [comment]
@cormacpayne Wow, long day, my apologies.

Here you go!

```none


    Directory: /usr/local/share/powershell/Modules


ModuleType Version    Name                                ExportedCommands                                                                 
---------- -------    ----                                ----------------                                                                 
Script     0.12.0     AzureRM.Netcore                                                                                                      
```

**Comment by maddieclayton** [comment]
@MiYanni Can you take a look at this issue?

**Comment by MiYanni** [comment]
@maddieclayton I cannot, unfortunately.

**Comment by markcowl** [comment]
@MiYanni There are two possibilities that come to mind:  one is a simple missing dependency - the type contverter is loaded through types.ps1xml, and is included in the profile cmdlets dll; two is a restricted environment that prevents loading type converters

**Comment by WernerMairl** [comment]
Same here!
- Install-Module without error
- Get-Module -Listavailable is empty => Import-Module fails

using the following docker file:

https://gist.github.com/WernerMairl/58ff2c59ba90c6bf02d74d980bd2143b

btw: the source code for YOUR docker image is available anywhere ? 
probably the source code helps me to find a workaround.....

regards
Werner

**Comment by MiYanni** [comment]
@jebalert This is fixed in 0.13. If you still have a problem, please refile an issue.
